src/task/ReceiveRemoteVideoPauseResumeTask.ts

```diff
@@ -36,6 +37,32 @@ export default class ReceiveRemoteVideoPauseResume
     this.context.removableObservers.push(this);
   }
 
+  updateSubscribedGroupdIds(groupIds: Set<number>): void {
+    const existingVideoTileIds = new Set<number>();
+    for (const groupId of groupIds) {
+      const attendeeId = this.context.videoStreamIndex.attendeeIdForGroupId(groupId);
+      if (attendeeId.length === 0) {
```

Comment:
Can `attendeeId` be `null` here if not found, sorry I dint check `attendeeIdForGroupId` implementation, so should a general falsy check be more appropriate here?

Reply:
It does not return `null` (even though other functions in that class do). Dreaming of strict mode :D. Just going to leave this as-is since its technically correct as long as no-one makes the return type of `attendeeIdForGroupId` inaccurate.